source/_components/device_tracker.geofency.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Geofency"
+description: "Instructions for how to use Geofency to track devices in Home Assistant."
+date: 2017-08-22 19:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: geofency.png
+ha_category: Presence Detection
+ha_release: 0.52
+---
+
+This platform allows you to detect presence using [Geofency](http://www.geofency.com/). Geofency is an [paid app](https://itunes.apple.com/app/id615538630) for iOS that allows users to configure a request that will be sent when a geofence or iBeacon region is entered or exited. This can be configured with Home Assistant to update your location.
```

Comment:
*Spelling*:
- `an paid` -> `a paid`

*Grammar*:
- `allows users to configure a request` -> `lets users configure a request` (allows is getting repetitive).